src/apps/tezos/get_address.py

```diff
@@ -4,17 +4,21 @@
 
 from apps.common import seed
 from apps.common.layout import show_address, show_qr
-from apps.tezos.helpers import TEZOS_CURVE, b58cencode, tezos_get_address_prefix
+from apps.tezos.helpers import (
+    TEZOS_CURVE,
+    TEZOS_ED25519_ADDRESS_PREFIX,
+    b58cencode,
+    tezos_get_address_prefix,
+)
 
 
 async def get_address(ctx, msg):
     address_n = msg.address_n or ()
     node = await seed.derive_node(ctx, address_n, TEZOS_CURVE)
 
-    sk = node.private_key()
-    pk = ed25519.publickey(sk)
+    pk = seed.remove_ed25519_prefix(node.public_key())
     pkh = hashlib.blake2b(pk, outlen=20).digest()
-    address = b58cencode(pkh, prefix=tezos_get_address_prefix(0))
+    address = b58cencode(pkh, prefix=TEZOS_ED25519_ADDRESS_PREFIX)
```

Comment:
Nice. Now we can use this also in `sign_tx._get_address_by_tag` and remove tezos_get_address_prefix completely.